api/templates/design.go

```diff
@@ -47,6 +47,7 @@ var _ = Service("templates", func() {
 	// CORS
 	cors.Origin("*", func() {
 		cors.Headers("Content-Type", "X-StorageApi-Token")
+		cors.Methods("GET", "POST", "PUT", "DELETE")
```

Comment:
![image](https://user-images.githubusercontent.com/19371734/168978438-85050b8a-84d4-49a0-9dfa-cd6fc811f2a7.png)

Reply:
To nemají defaultně povolený? 😕